community/pages/finder/Results/Profile/index.jsx

```diff
@@ -75,6 +76,17 @@ export default class Profile extends Component {
 
   }
 
+  // override view all results link if can go back
+  // this preserves scroll state
+  handleBack = (event) => {
+    const lastLocation = this.props.previousLocations[this.props.previousLocations.length-1];
+
+    if (lastLocation === `/groups/finder/list/${this.props.params.hash}`) {
+      event.preventDefault();
+      window.history.back();
```

Comment:
@johnthepink should we disptach this using our `{ routeActions } from "core/store/routing"`?